libogc/network_wii.c

```diff
@@ -885,40 +885,56 @@ s32 net_send(s32 s, const void *data, s32 size, u32 flags)
 s32 net_sendto(s32 s, const void *data, s32 len, u32 flags, struct sockaddr *to, socklen_t tolen)
 {
 	s32 ret;
+	const s32 maxblocksize = (NET_HEAP_SIZE-64);
```

Comment:
I would drop this to NET_HEAP_SIZE/2 to try and avoid failure in the net_mallocs.

I'd also try this as a static const at global scope & see what the code gen is like.

Reply:
moved to top of file :)